taca/utils/misc.py

```diff
@@ -197,7 +197,7 @@ def run_is_demuxed(run, couch_info=None):
     run_name = '{}_{}'.format(run_date, run_fc)
     try:
         couch_connection = statusdb.StatusdbSession(couch_info).connection
-        fc_db = couch_connection[couch_info['db']]
+        fc_db = couch_connection[couch_info['xten_db']]
```

Comment:
Both `db` and `xten_db` are `x_flowcells` in the taca.yaml. Do we need to differentiate between them anymore since we hardly use the original flowcells db?

Reply:
Hi, I added `db` this morning. I believe that all other functions of TACA are pointing to `xten_db`